db/migrate/20240910135453_add_journey_to_reminders.rb

```diff
@@ -0,0 +1,19 @@
+class AddJourneyToReminders < ActiveRecord::Migration[7.0]
+  def up
+    add_column :reminders, :journey, :string
+
+    # Additional payments journey was not open when migration created, however
+    # the FurtherEducationPayments journey was opened for the first time on
+    # 2024/9/9 and so some reminders may also be set for the
+    # FurtherEducationPayments journey
+    Reminder.not_yet_sent.where("created_at < ?", Date.new(2024, 9, 9)).update_all(journey: Journeys::AdditionalPaymentsForTeaching)
```

Comment:
Do we want to scope this by `not_yet_sent`? What about reminders in the DB that have been sent already?

Reply:
Good catch.